### PR TITLE
Fixes opacity on mwp sign txn sheet

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-1852

## What changed (plus any additional context for devs)
fixes opacity on route source for mwp

## Screen 
Screenshot 2024-09-06 at 10 35 23 AM
recordings / screenshots

## What to test
test that both mobile wallet protocol & wallet connect have a solid background on the SignTransaction sheet.

### CODE REVIEW

**Review by greg-schrammel — APPROVED** [review]
✌️

File: src/navigation/config.tsx [hunk 1]
```diff
@@ -277,7 +277,8 @@ export const signTransactionSheetConfig = {
   options: ({ route }: { route: SignTransactionSheetRouteProp }) => ({
     ...buildCoolModalConfig({
       ...route.params,
-      backgroundOpacity: route?.params?.source === RequestSource.WALLETCONNECT ? 1 : 0.7,
+      backgroundOpacity:
+        route?.params?.source === RequestSource.WALLETCONNECT || route?.params?.source === RequestSource.MOBILE_WALLET_PROTOCOL ? 1 : 0.7,
```
Comment (brunobar79) [inline src/navigation/config.tsx:281]:
```suggestion
        [RequestSource.WALLETCONNECT, RequestSource.MOBILE_WALLET_PROTOCOL].includes(route?.params?.source) ? 1 : 0.7
```

**Review by brunobar79 — CHANGES_REQUESTED** [review]
Could you please add PoW and testing instructions?

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/@matthew/APP-1852/cd5af652b1e8c86a9af8bda405af53a2e2e5fa8e.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/@matthew/APP-1852/cd5af652b1e8c86a9af8bda405af53a2e2e5fa8e.ipa) for cd5af652b1e8c86a9af8bda405af53a2e2e5fa8e

**Review by benisgold — APPROVED** [review]

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/@matthew/APP-1852/ca81103215d3ace6c4f15a029e7d79312363c539.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/@matthew/APP-1852/ca81103215d3ace6c4f15a029e7d79312363c539.ipa) for ca81103215d3ace6c4f15a029e7d79312363c539

**Comment by jinchung** [comment]
adding source convo: https://rainbowhaus.slack.com/archives/C02C2FVC6N6/p1725561738782969

**Pull request merged**